src/components/User.js

```diff
@@ -26,13 +36,21 @@ class User extends Component {
   render() {
     console.log(this.props);
     if (this.state.loading) { return (<Load />); };
-    return (
-      <Row>
-        {this.state.data.items.map((pullrequest, idx) => {
-          return <PullRequest data={pullrequest} key={idx} />;
-        })}
-      </Row>
-    );
+    const { items } = this.state.data
```

Comment:
Codacy is complaining about this line.
https://www.codacy.com/app/ducthienbui97/pullrequests/pullRequest?prid=957551